src/modules/options/OptionsWidget_interfaceFeatures.cpp

```diff
@@ -53,14 +53,23 @@ OptionsWidget_interfaceFeatures::OptionsWidget_interfaceFeatures(QWidget * paren
 #else
 	addBoolSelector(0,5,0,5,__tr2qs_ctx("Require Ctrl to be held down to copy text","options"),KviOption_boolRequireControlToCopy);
 #endif
+#ifdef COMPILE_ON_WINDOWS
```

Comment:
un1versal, it's not defined on mingw?

2016-04-25 14:34 GMT+01:00 un1versal <notifications@github.com>:

> In src/modules/options/OptionsWidget_interfaceFeatures.cpp
> <https://github.com/kvirc/KVIrc/pull/1942#discussion_r60912812>:
>
> > @@ -53,14 +53,23 @@ OptionsWidget_interfaceFeatures::OptionsWidget_interfaceFeatures(QWidget * paren
> >  #else
> >  	addBoolSelector(0,5,0,5,__tr2qs_ctx("Require Ctrl to be held down to copy text","options"),KviOption_boolRequireControlToCopy);
> >  #endif
> > +#ifdef COMPILE_ON_WINDOWS
>
> and again
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kvirc/KVIrc/pull/1942/files/b011f1d8681f5343fbd8f6e02aceae191b4e49f6#r60912812>
>